test/python/WMCore_t/WMRuntime_t/Tools_t/Scram_t.py

```diff
@@ -107,7 +107,10 @@ def testC(self):
     def testArchMap(self):
         self.assertItemsEqual(OS_TO_ARCH['rhel6'], ['slc5', 'slc6'])
         self.assertItemsEqual(OS_TO_ARCH['rhel7'], ['slc7'])
+        self.assertItemsEqual(OS_TO_ARCH['rhel8'], ['cc8', 'cs8', 'alma8'])
```

Comment:
I don't know if this is generic enough, I rather think you should apply sort to final list to get consistent results all the time. But this should be do to all checked lists (this and on line 108 and may be others if they exist in a code). It really depends on a code which make these lists.

Reply:
I had your very same worry about the sorting of lists in tests when I was doing the py3 migration.

I think that Alan's changes are as safe as possible. Inside the [doc](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertItemsEqual) it is specified that this test already sorts internally the two lists. Yes, this was renamed in python3. Yes, i think that [assertCountEqual](https://docs.python.org/3.10/library/unittest.html#unittest.TestCase.assertCountEqual) is even a worse name for such an assertion. 

I am sorry for intruding here, Stefano asked me to adapt CRAB code to use what you will develop in this PR and stumbled upon this comment.

Reply:
Thanks, Dario. Yes, Valentin. In short, we do need to sort these lists to run a comparison.